mne/io/edf/edf.py

```diff
@@ -233,14 +246,18 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     stim[n_start:n_stop] += evid
                 data[stim_channel_idx, :] = stim[start:stop]
             else:
-                # Allows support for up to 17-bit trigger values (2 ** 17 - 1)
                 stim = np.bitwise_and(data[stim_channel_idx].astype(int),
-                                      131071)
+                                      2**17 - 1)
+
+                if biosemi:
+                    stim = stim >> 8  # not sure why but it works
```

Comment:
I found this:


> Bit 16: High when new Epoch is started
Bit 17: Speed bit 0
Bit 18: Speed bit 1
Bit 19: Speed bit 2
Bit 20: High when CMS is within range
Bit 21: Speed bit 3
Bit 22: High when battery is low
Bit 23: (MSB) High if ActiveTwo MK2

Here:
https://www.biosemi.com/faq/trigger_signals.htm

I'd say we can safely skip this. Maybe Bit16 could be useful, but I've been using a Biosemi device for years and have no idea how to specify these "Epochs"